CHANGELOG.md

```diff
@@ -1,6 +1,73 @@
 # Changelog
 ## Unreleased
 
+## v0.113.0
+
+This Splunk OpenTelemetry Collector release includes changes from the opentelemetry-collector v0.113.0 and the opentelemetry-collector-contrib v0.113.0 releases where appropriate.
+
+### 🛑 Breaking changes 🛑
+
+- (Contrib) `sapmreceiver`: Remove the deprecated access_token_passthrough from SAPM receiver. ([#35972](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35972))
+  Please use `include_metadata` instead with the following config option applied to the batch processor:
+  batch:
+    metadata_keys: [X-Sf-Token]
+
+- (Contrib) `pkg/ottl`: Promote `processor.transform.ConvertBetweenSumAndGaugeMetricContext` feature gate to Stable ([#36216](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36216))
+  This gate can no longer be disabled. The `convert_sum_to_gauge` and `convert_gauge_to_sum` may now only be used with the `metric` context.
+
+
+### 💡 Enhancements 💡
+
+- (Contrib) `splunkenterprisereceiver`: Add telemetry around the Splunk Enterprise kv-store. ([#35445](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35445))
+- (Contrib) `journaldreceiver`: adds ability to parse journald's MESSAGE field as a string if desired ([#36005](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36005))
+- (Contrib) `journaldreceiver`: allows querying a journald namespace ([#36031](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36031))
+- (Contrib) `hostmetricsreceiver`: Add the system.uptime metric in the hostmetrics receiver ([#31627](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31627))
+  This metric is provided by the new `system` scraper.
+
+- (Contrib) `hostmetrics`: Adjust scraper creation to make it so the scraper name is reported with hostmetrics scraper errors. ([#35814](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35814))
+- (Contrib) `pkg/translator/prometheusremotewrite`: `FromMetricsV2` now transforms attributes into labels. ([#33661](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33661))
```

Comment:
We don't use this module, so don't need to mention this change

Reply:
Ah, I thought `signalfxgatewayprometheusremotewrite` might be. But doesn't look like it.